packages/xml-body-parser/src/index.spec.ts

```diff
@@ -573,22 +597,6 @@ describe('XmlBodyParser', () => {
                 CreatedAt: new Date(isoString)
             });
         });
-
```

Comment:
Did you mean to drop these tests?

Reply:
I delete them unintentionally. Will add them back